py/tests/integration/setup.yaml

```diff
@@ -112,7 +112,7 @@ tests: # test specific settings
         amount: 20
         fee: 1
       create_contract:
```

Comment:
In this commit, [workflow with Python UAT tests](https://circleci.com/workflow-run/1cec3430-cca4-4cc3-b12b-bad0b862f0e0).